themebuilder-scss/package.json

```diff
@@ -29,13 +31,20 @@
     "@types/clean-css": "^4.2.1",
     "@types/jest": "^25.2.1",
     "@types/node-sass": "^4.11.0",
-    "eslint": "^6.8.0",
+    "@typescript-eslint/eslint-plugin": "^2.33.0",
+    "eslint": "^7.0.0",
+    "eslint-config-airbnb-typescript": "^7.2.1",
```

Comment:
Why do we need this package?

Reply:
eslint asks to install it when we use eslint-config-devextreme

Reply:
Ok. Let's continue our discussion here:
https://github.com/DevExpress/DevExtreme/pull/13036#discussion_r427837333